code/modules/client/preference_setup/loadout/loadout_suit.dm

```diff
@@ -54,6 +54,7 @@
 /datum/gear/suit/hazard_vest
 	display_name = "hazard vest selection"
 	path = /obj/item/clothing/suit/storage/hazardvest
+	cost = 1
```

Comment:
A good reason why they cost 2 is because most of these provide storage in the form of pockets, like webbing, not including the suit storage being able to store things too.

Reply:
I changed that line in an older version of this PR, which I am updating and removing parts of